canonical-websites/www.canonical.com
# Convert /about to Vanilla v1 styles

## Done

Upgrade /about to v1

## QA

- Pull code
- Check markup at `templates/about/index.html`
- Run `./run serve --watch`
- Check page locally at http://0.0.0.0:8002/about

Fixes https://www.github.com/ubuntudesign/vanilla-squad/issues/28

## Demo

http://www.canonical.com-homepage-polish.demo.haus/about
